autoload/test/python/pyunit.vim

```diff
@@ -39,7 +39,7 @@ function! test#python#pyunit#executable() abort
     let pipenv_prefix = "pipenv run "
   endif
 
-  return pipenv_prefix . 'python -m unittest'
+  return pipenv_prefix . 'python3 -m unittest'
```

Comment:
Instead of hard coding this use detection, for example

```viml
if strlen(system('command -v python3'))
  return pipenv_prefix . 'python3 -m unittest'
else
  return pipenv_prefix . 'python -m unittest'
endif
```